resources/ory/charts/hydra/templates/service-admin.yaml

```diff
@@ -16,12 +21,42 @@ spec:
     - port: {{ .Values.service.admin.port }}
       targetPort: http-admin
       protocol: TCP
-      name: http
-    - port: 15020
-      targetPort: 15020
-      protocol: TCP
-      name: tcp-status-port
+      name: {{ .Values.service.admin.name }}
   selector:
     app.kubernetes.io/name: {{ include "hydra.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
+{{- if and (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") (.Values.serviceMonitor.enabled) }}
```

Comment:
Do we need this service? Isn't this already done in Kyma/monitoring component?

Reply:
I think service monitor is still needed here

Reply:
Correction, they are placed in ory/templates, so we can remove servicemonitors